.github/workflows/main.yml

```diff
@@ -60,7 +60,7 @@ jobs:
       - name: Build
         run: ./build.sh --release
       - name: Run tests
-        run: cargo test
+        run: cargo test -- --nocapture
```

Comment:
Is this necessary?

Reply:
I can remove this. I was debugging an error that I couldn't reproduce on the local machine, so I turned `--nocapture` on. Otherwise, cargo test will not print failed error message.

Reply:
Actually, this seems to be the solution to #150 LOL. Without `--nocapture`, cargo test hides everything.